mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-test-groups-tags.asciidoc

```diff
@@ -1,3 +1,20 @@
+:toc: macro
+
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+toc::[]
+:idprefix:
+:idseparator: -
+:reproducible:
+:source-highlighter: rouge
+:listing-caption: Listing
+
 = Why join Test Cases in groups - Test Suites
 
```

Comment:
Please update this file and remove all "Prior to mrchecker-core-module version 5.6.2.1:" parts